demo-extended/templates/deployment.yaml

```diff
@@ -17,32 +18,50 @@ spec:
       containers:
         - name: {{ .Values.name }}
           image: "{{ .Values.general.image }}:{{ .Values.general.version }}"
-          {{- if ((.Values.networking).expose) }}
+          
+          {{- if .Values.networking.expose }}
           ports:
             - name: http
-              containerPort: {{ .Values.networking.port | default 80 }}
+              containerPort: {{ .Values.networking.port }}
               protocol: TCP
           {{- end }}
+          
           env:
-          {{- if ((.Values.general).environment) }}
-          {{- range $key, $value := (.Values.general).environment }}
-            - name: {{ $key }}
-              value: {{ $value | quote }}
+          {{- if .Values.general.environment }}
+          {{- toYaml .Values.general.environment | indent 12 }}
```

Comment:
Can you revert this? With the UI for env variables, the `indent` doesn't work
<img width="586" alt="Screenshot 2024-05-04 at 22 42 24" src="https://github.com/cyclops-ui/templates/assets/72022639/72e24621-48b1-431b-ab73-ffab5a1a2ca4">